examples/04-Exporting-ranking-models.ipynb

```diff
@@ -70,9 +70,9 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !apt-get install tree
</pre><p>Can you pls  remove these lines and instead add  <code>!pip install seedir</code>?   Seedir is a python package for creating, editing, and reading folder tree diagrams. Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>